backend/src/main/java/com/example/backend/security/UserController.java

```diff
@@ -66,6 +66,11 @@ public void removeCardFromFavorites(Principal principal, @PathVariable String ca
         userService.removeCardFromFavorites(cardId, principal.getName());
     }
 
+    @GetMapping("/all")
+    public List<MongoUser> getAllUsers(){
+        return userService.getAllUsers();
+    }
+
```

Comment:
Watch out you publish all the users passwords and all other sensetive data here!